README.md

```diff
@@ -11,7 +11,7 @@ Each "page" of this story is a separate file.  The files are Markdown, which
 means they are plain text, but with a few special capabilities.  The only
 Markdown capability used in this story is links.  By putting text in brackets
 and a URL in parentheses, you can add a link to the story.  For example, the
-following text would turn "here" into a link to Google: "Click
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.